src/utils/state-config.js

```diff
@@ -84,9 +84,9 @@ class StateConfig {
     const config = this.all
 
     if (args.length === 1) {
-      for (const keyPart of Object.keys(key)) {
+      Object.keys(key).forEach((keyPart) => {
```

Comment:
How about using `Object.entries`?

Reply:
Yeah. `Object.entries` seems more suitable now that the loop has been removed.